src/containers/CounterButton.js

```diff
@@ -0,0 +1,25 @@
+import { connect } from 'react-redux';
+import {increaseCounter} from "../actions";
+import {decreaseCounter} from "../actions";
+import CounterButton from "../components/CounterButton";
+
+function mapDispatchToProps(dispatch){
+  return {
+   incresase:function(txt){
+     let action = increaseCounter(txt);
+     dispatch(action);
+   }
+ }
+ }
+ function mapDispatchToProps(dispatch){
+   return {
+
+   decrease:function(txt){
```

Comment:
move the decrease key and value into the object where increase already is. No need for two mapDispatchToProps